src/test/java/seedu/duke/storage/StorageTest.java

```diff
@@ -0,0 +1,96 @@
+package seedu.duke.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import seedu.duke.entries.Category;
+import seedu.duke.entries.Entry;
+import seedu.duke.entrylog.EntryLog;
+
+import java.util.List;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.io.File;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@DisplayName("Test Storage")
+public class StorageTest {
+    private final String pathString = "./test/storage.txt";
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+    private final PrintStream originalOut = System.out;
+    private final PrintStream originalErr = System.err;
+
+    @BeforeEach
+    void init() {
+        // setup streams
+        System.setOut(new PrintStream(outContent));
+        System.setErr(new PrintStream(errContent));
+    }
+
+    @AfterEach
+    void teardown() {
+        // restore streams
+        System.setOut(originalOut);
+        System.setErr(originalErr);
+
+    }
+
```

Comment:
These methods are not required since there is no testing of data in the output stream.